src/pages/Vai/RepayVai/index.tsx

```diff
@@ -117,6 +120,8 @@ export const RepayVaiUi: React.FC<IRepayVaiUiProps> = ({
                   >
                     {readableRepayableVai}
                   </LabeledInlineContent>
+
```

Comment:
On line 117, the `isLast` prop needs to be set to `false` since it's to the last row anymore

Reply:
Done, thanks!